CHANGELOG.md

```diff
@@ -2,6 +2,63 @@
 
 All notable changes to the kdb extension will be documented in this file.
 
+
+# v1.3.0
+
+### Enhancements
+
+- Ability to connect to kdb Insights Enterprise using a WSL connection.
+- Execute files from the right-click menu.
+
```

Comment:
on Results tab, hover the mouse at the column, show the column Q type if it exist

Reply:
only for insights connections

Reply:
- On the Results tab, on a kdb Insights Enterprise connection, hover the mouse over a column header to see  the column Q type if it exists.

Reply:
```suggestion
- On the Results tab, on a kdb Insights Enterprise connection, hover the mouse over a column header to see the column Q type if it exists.
```